examples/pitr/main.tf

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# ICD postgresql database
+##############################################################################
+
+module "postgresql_db" {
+  source            = "../.."
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-postgres"
+  region            = var.region
+  resource_tags     = var.resource_tags
+  configuration     = var.configuration
+}
+
+resource "null_resource" "wait_for_postgresql_db_pitr_creation" {
```

Comment:
could we use native tf support for the sleep here: https://registry.terraform.io/providers/hashicorp/time/latest/docs/resources/sleep